addons/mail/models/discuss/discuss_channel.py

```diff
@@ -1116,11 +1119,13 @@ def _get_last_messages(self):
 
     def _load_more_members(self, known_member_ids):
         self.ensure_one()
-        unknown_members = self.env['discuss.channel.member'].search(
+        # sudo: discuss.channel.member - reading non-self members requires sudo to avoid ACL issues
```

Comment:
These comments should say why it is acceptable functionally, not why we need to do it technically.

In this case: reading xxx fields of other members from common channel is acceptable.
Note: the default _to_store of member is actually returning fields by default that shouldn't be allowed to be seen by other users, such as `last_seen_dt`

When we use sudo, it would probably be better to list the fields we want to send explicitly.